include/aspect/particle/property/function.h

```diff
@@ -69,6 +69,18 @@ namespace aspect
                                             std::vector<double> &particle_properties) const;
 
           /**
+           * Returns an enum, which determines how this particle property is
+           * initialized for particles that are created later than the initial
+           * particle generation, e.g. to balance the particle load or prevent
+           * empty cells. The current implementation returns
```

Comment:
With "current implementation", do you mean the "implementation of this function in the current class"? I think the latter is clearer. Same in the parallel classes.